docs/Makefile

```diff
@@ -49,12 +49,21 @@ help:
 	@echo "  coverage   to run coverage check of the documentation (if enabled)"
 
 clean:
-	rm -rf $(BUILDDIR)/*
+	@rm -rf $(BUILDDIR)/*
 
-html:
+clean_rst: clean
+	@rm -rf built_rst/csv/elements*
+	@rm -rf built_rst/xml/elements*
+
+rst: clean_rst
+	@cd .. && python scripts/vip.py norm_yaml
+	@cd .. && python scripts/vip.py make_rest
+
+html: rst
 	$(SPHINXBUILD) -b html $(ALLSPHINXOPTS) $(BUILDDIR)/html
 	@echo
 	@echo "Build finished. The HTML pages are in $(BUILDDIR)/html."
+	@sphinx-autobuild . _build/html
```

Comment:
Now the `html` command does the whole chain:

- Removes old RST files
- Builds new ones
- Builds HTML
- starts up the server for review

Reply:
suggestion, non-blocking

is this the "dev" setup, basically? as in, if i'm a developer who wants to tweak something, would i make my changes and run `make html` to preview them?

if so, thoughts on making a `make dev` alias for `html` so this is more clear?